ansible/tasks/setup-system.yml

```diff
@@ -131,3 +131,18 @@
     mode: 0644
     owner: root
     group: root
+
+#Set Sysctl params for restarting the OS on oom after 10
+- name: Set vm.panic_on_oom=1
+  ansible.builtin.sysctl:
+    name: vm.panic_on_oom
+    value: '1'
```

Comment:
Do we want 2? I guess 1 is fine right now since PG memory usage is not capped?

Reply:
I thought 1 seemed better sense we aren't using cgroups.

Reply:
Some of our services use systemd resource controls (e.g. https://github.com/supabase/postgres/blob/develop/ansible/files/gotrue.service.j2#L13) which internally uses cgroups afaik